core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -209,6 +224,40 @@ private ReadWriteLock getPageLock(PageId pageId) {
   @Override
   public boolean put(PageId pageId, byte[] page) {
     LOG.debug("put({},{} bytes) enters", pageId, page.length);
+    if (!mAsyncWrite) {
+      boolean inserted = putInternal(pageId, page);
+      if (inserted) {
+        LOG.debug("put({},{} bytes) exits", pageId, page.length);
+      } else {
+        LOG.debug("put({},{} bytes) fails", pageId, page.length);
+      }
```

Comment:
Simplify this by logging a succeeded field?

Reply:
done